src/MongoDB/Query.c

```diff
@@ -98,7 +107,7 @@ static bool php_phongo_query_opts_append_document(bson_t* opts, const char* opts
 	}
 
 /* Note: handling of integer options will depend on SIZEOF_ZEND_LONG and we
-	 * are not converting strings to 64-bit integers for 32-bit platforms. */
+ * are not converting strings to 64-bit integers for 32-bit platforms. */
```

Comment:
Was this introduced by clang-format and fixed manually?

Reply:
No, clang had ignored it because it was a tab - so I fixed it by using a space.